docs/resources/server/user.md

```diff
@@ -1 +1,35 @@
 # User Resource
+| Key | Name/Value | Type | Description |
+| --- | ---------- | ---- | ----------- |
+| 1 | Username | **string** | The name of player |
+| 2 | Userid | **number** | The ID of player |
+| 3 | Stars | **number** | The count of stars player have |
+| 4 | Demons | **number** | The count of demons player have |
```

Comment:
6 should be added as another unknown since it does exist

Reply:
note that 6 is not always returned, this should be mentioned in the documentation as well

Reply:
update: 6 is the global leaderboard position, it's only returned by getGJScores.php with type "relative"